config.lua.dist

```diff
@@ -76,6 +76,12 @@ classicEquipmentSlots = false
 classicAttackSpeed = false
 showScriptsLogInConsole = true
 
+-- Server Save
+notifyServerSave = true
+cleanMapAtServerSave = false
+closeAtServerSave = false
+shutdownAtServerSave = true -- If you set close and shutdown to 'true' it will close and then shutdown the server.
```

Comment:
This comment doesn't really say anything more than the variable names already say